core/components/atoms/_chip/__tests__/_chip.test.tsx

```diff
@@ -44,7 +44,7 @@ describe('Chip component', () => {
 
   it('renders chip component with minimum props', () => {
     const { getByTestId, queryByTestId } = render(<GenericChip label="ChipLabel" name="Chip" />);
-    expect(getByTestId('DesignSystem-GenericChip--GenericChipWrapper')).toBeInTheDocument();
+    expect(getByTestId('DesignSystem-GenericChip--Wrapper')).toBeInTheDocument();
```

Comment:
Please update this data-test `DesignSystem-GenericChip--Wrapper`, Wrapper does not work as a modifier here